src/eradiate/scenes/atmosphere/_particle_layer.py

```diff
@@ -154,26 +153,27 @@ def _bottom_top_validator(self, attribute, value):
         default="16",
     )
 
-    # TODO: replace with actual data set and load through data interface
-    # TODO: change defaults to production data
-    dataset: pathlib.Path = documented(
+    dataset: xr.Dataset = documented(
         attr.ib(
-            default=path_resolver.resolve("tests/radprops/rtmom_aeronet_desert.nc"),
-            converter=path_resolver.resolve,
-            validator=[attr.validators.instance_of(pathlib.Path), validators.is_file],
+            default="tests/radprops/rtmom_aeronet_desert.nc",
+            converter=converters.load_dataset,
+            validator=attr.validators.instance_of(xr.Dataset),
         ),
-        doc="Path to particle radiative property data set. Defaults to testing data.",
-        type="Path",
-        init_type="path-like",
+        doc="Particle radiative property data set. If a path is passed, the "
+        "converter tries to open the corresponding file on the hard drive; if "
+        "this fails, it tries to load a resource from the data store.",
+        type="Dataset",
+        init_type="Dataset or path-like, optional",
+        default="tests/radprops/rtmom_aeronet_desert.nc",
```

Comment:
I agree with you that this default pointing to a test data set is weird.
The continental particles look like an appropriate default to me:

```suggestion
        default="spectra/particles/govaerts_2021-continental-extrapolated.nc",
```